spec/features/navbar_spec.rb

```diff
@@ -1,66 +0,0 @@
-require 'rails_helper'
```

Comment:
Why did you remove the entire navbar specs?

Reply:
Because the whole navbar changed (non of the tests are relevant for the new navbar) and I wouldn't test the frontend functionality

Reply:
But I'll just rewrite the test in order to test, if contacts etc. are not shown if not signed in